src/css/Header.module.css

```diff
@@ -0,0 +1,47 @@
+.wrapper {
+  height: 100%;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+}
+
+.backgroundContainer {
+  background-image: url(../assets/frontpage-background.svg);
+  background-size: cover;
+  height: 475px;
+}
+
+.logo {
+  margin-top: 50px;
+}
+
+.headerText {
+  color: var(--color-main);
+  justify-content: center;
+  text-align: center;
+}
+
+.mandate {
+  font-size: var(--fontSize-mobile-subHeading);
+  font-size: 20px;
+}
+
+.mission {
+  width: 75%;
+  margin: auto;
+  text-align: left;
+  font-size: 16px;
+}
+
+.btn {
+  margin-top: 20px;
+  display: flex;
+  justify-content: space-evenly;
```

Comment:
This isn't really what happing in the designs, I would recommend you set `justify-content: space-between;`, `margin: 20px auto 0 auto` (replace `margin-top`) and give this container the same width as the `.mission`.